src/main_cordova.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Betaflight configurator</title>
+    <style>
+        * {
```

Comment:
I see a lot of CSS and JS included inside the HTML. Is this needed with Cordova? It has some benefits? Until now, we prefer to split this to it's own CSS and JS.

Reply:
Cordova is dependent of an app called 'Webview' natively installed on Android devices. `main_cordova.html` is used to check if the Webview app version is compatible with the configurator and if it's not the case, will help the user to update this Webview app (in fact, it could be a difficult task to update webview)

Reply:
I will split it to it's own CSS and JS